src/deployment/default/colors.ts

```diff
@@ -1,142 +1,14 @@
+/**
+ * (c) 2021, Micro:bit Educational Foundation and contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
 import { theme } from "@chakra-ui/theme";
 
-const gray = {
-  10: "#fcfcfc",
-  25: "#f5f5f5",
```

Comment:
Might need to remove the use of "gray.25" and change "gray.25" -> "gray.100" in GestureNameGridItem.tsx so that the text input can be seen
<img width="499" alt="Screenshot 2024-10-03 at 12 14 13" src="https://github.com/user-attachments/assets/28fc20f7-febf-4c82-910c-20640e9c49ab">

Reply:
Good spot, thanks.

Reply:
Although I'd like to rationalise them I'm just pulling them into the default theme

Reply:
4e0aef909134d4a330cace6d1bd3cfa415f1114d